api/src/main/java/uk/gov/dhsc/htbhf/claimant/model/eligibility/PersonDTO.java

```diff
@@ -0,0 +1,34 @@
+package uk.gov.dhsc.htbhf.claimant.model.eligibility;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import uk.gov.dhsc.htbhf.claimant.model.AddressDTO;
+
+import java.time.LocalDate;
+
+/**
+ * Representation of what is required to  call the Eligibility Service.
+ */
+@Data
+@Builder
+@AllArgsConstructor(onConstructor_ = {@JsonCreator})
+public class PersonDTO {
```

Comment:
We specifically didn't add validation annotations to this class as its never validated - we might want to do the same thing to the `DWPPersonDTO` in the eligibility service.